cmd/thanos/main.go

```diff
@@ -212,6 +212,38 @@ func registerMetrics(mux *http.ServeMux, g prometheus.Gatherer) {
 	mux.Handle("/metrics", promhttp.HandlerFor(g, promhttp.HandlerOpts{}))
 }
 
+func registerHealthy(mux *http.ServeMux, logger log.Logger, component string) {
+	mux.HandleFunc("/-/healthy", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
```

Comment:
Let's delete this. This is not needed if we wrtie below `Thanos is healthy` succesfully.

See `WriteHeader` comment: 
> If WriteHeader is not called explicitly, the first call to Write
	// will trigger an implicit WriteHeader(http.StatusOK).

This is also wrong, as we write header second time if Fprintf fails - this will most likely produce panic